server/common/utils/index.js

```diff
@@ -51,11 +51,30 @@ const responseWithItem = (item, userId) => {
 };
 
 const responseWithCohorts = (cohorts, userId) =>
-  _map(cohorts, ({ favIds, ...rest }) => ({
-    ...rest._doc,
-    isFavourite: isUserFavourite(favIds, userId)
+  _map(cohorts, ({ _doc }) => {
+    const { favIds, ...rest } = _doc;
+    return {
+      ...rest,
+      isFavourite: isUserFavourite(favIds, userId)
+    };
+  });
+
+const responseWithUsers = (users, ownerIds) =>
+  users.map(user => ({
+    ...user._doc,
+    isOwner: ownerIds.indexOf(user._doc._id.toString()) > -1
   }));
 
+const responseWithUser = data => {
+  const { avatarUrl, displayName, newMemberId } = data;
+  return {
+    _id: newMemberId,
+    avatarUrl,
+    displayName,
+    isOwner: false
```

Comment:
why is this always `false`?

Reply:
because when adding a new member to cohort, he is NOT the owner by default.